client/components/overview/AddToCart.jsx

```diff
@@ -1,53 +1,129 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import map from 'lodash/map';
+import range from 'lodash/range';
 import { CartContainer, CartButton, DropDown } from './styles/AddToCart.style';
 
-const AddToCart = ({ styles }) => (
-  <CartContainer
-    data-testid="cart-container"
-    className="cart-container"
-  >
-    <CartButton
-      data-testid="add-to-cart-button"
-      className="add-to-cart-button"
-      type="submit"
-    >
-      Add To Cart
-    </CartButton>
-
-    <DropDown
-      data-testid="size"
-      className="size"
-      name="size"
-      id="size"
-    >
-      {styles.results.map((style) => (
-        map(style.skus, (sku) => (
-          // does not yet reflect corresponding size/quantity data
-          <option value={`${sku}`}>{sku.size}</option>
-        ))
-      ))}
-    </DropDown>
-    <DropDown
-      data-testid="quantity"
-      className="quantity"
-      name="quantity"
-      id="quantity"
+const AddToCart = ({ addProductToCart, currentStyle }) => {
+  const [currentSize, setCurrentSize] = useState('Size');
+  const [currentQuantity, setCurrentQuantity] = useState('Quantity');
+  const [currentSku, setCurrentSku] = useState(null);
+  const [quantityForSelectedSize, setQuantityForSelectedSize] = useState(null);
+  const [cartIsDisabled, setCartIsDisabled] = useState(true);
+  const [quantityIsDisabled, setQuantityIsDisabled] = useState(true);
+
+  const handleSizeChange = (event) => {
+    const { size, sku, quantity } = JSON.parse(event.target.value);
+
+    setCurrentSize(size);
+    setCurrentSku(sku);
+    setQuantityForSelectedSize(quantity);
+    setQuantityIsDisabled(false);
+  };
+
+  const handleQtyChange = (event) => {
+    const { value } = event.target;
+
+    setCurrentQuantity(value);
```

Comment:
I think I do see a bug here where the first option of the select dropdown will change to whatever option was last selected. I don't think this is a breaking change so this can be addressed later